pkg/apis/resource/resource.go

```diff
@@ -22,22 +22,15 @@ import (
 	"github.com/tektoncd/pipeline/pkg/apis/pipeline"
 	pipelinev1beta1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
 	resourcev1alpha1 "github.com/tektoncd/pipeline/pkg/apis/resource/v1alpha1"
-	"github.com/tektoncd/pipeline/pkg/apis/resource/v1alpha1/git"
 	"github.com/tektoncd/pipeline/pkg/apis/resource/v1alpha1/image"
-	"github.com/tektoncd/pipeline/pkg/apis/resource/v1alpha1/storage"
 )
 
 // FromType returns an instance of the correct PipelineResource object type which can be
 // used to add input and output containers as well as volumes to a TaskRun's pod in order to realize
 // a PipelineResource in a pod.
 func FromType(name string, r *resourcev1alpha1.PipelineResource, images pipeline.Images) (pipelinev1beta1.PipelineResourceInterface, error) {
-	switch r.Spec.Type {
-	case resourcev1alpha1.PipelineResourceTypeGit:
-		return git.NewResource(name, images.GitImage, r)
```

Comment:
Does removing this mean we can remove all references to `GitImage`, including the `--git-image` flag passed to the controller in config/controller.yaml?

Reply:
Thanks @imjasonh .  We might want to move them to catalog and leave it here for now according to the updated plans at [tep074](https://github.com/tektoncd/community/commit/5f65433ebd7869dcfbd8c4f0db2d0cfe6331d8c6)